PACKAGES.md

```diff
@@ -13,7 +13,12 @@
 # Ui Components
 
 - [v-tooltip](https://github.com/Akryum/vue-tooltip), vue:2.0, links:[demo](https://akryum.github.io/vue-tooltip/), status:dev
+- [vue-multiselect](https://github.com/monterail/vue-multiselect), vue:1.0|2.0, links:[demo](http://monterail.github.io/vue-multiselect/), status:stable
```

Comment:
You may want to add the [doc link](http://monterail.github.io/vue-multiselect/#sub-getting-started) as well before the demo link.